valve-monitor/web-app/src/styles/Header.module.scss

```diff
@@ -3,6 +3,7 @@
 .header {
   display: flex;
   justify-content: space-between;
+  padding: 0 32px !important;
```

Comment:
I believe you can use `:global .ant-layout-header` to avoid `!important`.

Reply:
This `:global .ant-xyz` is only supported inside of some sort of a CSS class wrapper, if it's a top level component, the CSS module freaks out. To avoid breaking the rest of the header CSS that hinges on `.header` being the top level CSS class, I'm going to leave as is